_includes/notify_form.html

```diff
@@ -0,0 +1,18 @@
+{% assign form = site.data.notify-form %}
+
+<form class="notify-form form-inline">
```

Comment:
BTW whenever the button of submit stays below the input the button and the input should be width 100% as two blocks